server/database.py

```diff
@@ -10,43 +12,85 @@ def connect_to_db(config):
 def store_game(db_connection, game_data):
```

Comment:
This function's a little big overall. We could probably split it into storing the blocks and storing the actions as two separate functions.